conf/dkfz.config

```diff
@@ -0,0 +1,30 @@
+
+params {
+    config_profile_contact     = 'Kübra Narcı kuebra.narci@dkfz-heidelberg.de'
+    config_profile_name        = 'DKFZ cluster'
+
+    max_cpus   = 30
+    max_memory = '250.GB'
+    max_time   = '48.h'
+}
+
+
+singularity {
+  enabled = true
+  cacheDir = "/omics/groups/OE0246/shared/nf_cache_dir"
```

Comment:
Does this hold true for the entire DKFZ or only your group?

Reply:
this is a shared directory for OE0246.

Reply:
is `OE0246` for the entirety of DKFZ or just your group?

Reply:
that is a group but I am not sure if we can open a shared directory for all DKFZ

Reply:
Can you then specify the config to highlight its just for your group at DKFZ? e.g. `dkfz_biocore` or whatever the group is called? That way others from DKFZ could copy its content and make their own configs if necessary in the future.

Reply:
but anyone if dkfz can use this config only by chancing cacheDir, ist ok to only add a note there?

Reply:
We have the tendency in previous examples (e.g. check the QBIC configs), to specify multiple configs for groups then if possible - e.g. QBIC belongs to university of tuebingen, but they never made a config for Tübingen University as the general approaches differ to much between research groups / institutes there - so everyone there needs to build their own. Same would apply here, so make "your" config work for your group, but then do not name it only dkfz as you're not dkfz alone ;-) 

dkfz_kuebras_research_group (or some other appropriate abbreviation maybe?)